src/main/java/randoop/output/MinimizerWriter.java

```diff
@@ -35,8 +36,15 @@ public File writeClassCode(String packageName, String classname, String classCod
     File testFile = javaFileWriter.writeClassCode(packageName, classname, classCode);
 
     // Minimize the error-revealing test that has been output.
-    Minimize.mainMinimize(
-        testFile, Minimize.suiteclasspath, Minimize.testsuitetimeout, Minimize.verboseminimizer);
+    try {
+      Minimize.mainMinimize(
+          testFile, Minimize.suiteclasspath, Minimize.testsuitetimeout, Minimize.verboseminimizer);
+    } catch (IOException e) {
+      System.err.println("IOException: " + e.getMessage());
```

Comment:
The printed stack trace on the next line contains the message and the type, so this line isn't neede.